scripts/jest.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+set -ex
+
+NODE_ENV=test
+JEST_JUNIT_OUTPUT=reports/junit/js-test-results.xml
+
+node --expose-gc --max_old_space_size=4096 ./node_modules/.bin/jest --no-cache --runInBand --logHeapUsage --config jest.config.v1.js
+node --expose-gc --max_old_space_size=4096 ./node_modules/.bin/jest --no-cache --runInBand --logHeapUsage --config jest.config.v2.js
```

Comment:
In this file we need to introduce an `if` conditional that will split on the `v1/v2` flag thrown in from CI:
```
yarn test:ci:jest --v1
```